docs/content/02-tutorials/07-custom-domain-name.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "g. Custom Domain Name"
+weight: 27
+---
+
+*Custom domain names* are simpler and more intuitive URLs that you can provide to your API users.
+
+When deploying PCM, you will be able to invoke APIs using the default base URL, which is composed of an ID generated by API gateway, the region and the stage name. For example:
+
+> `https://api-id.execute-api.region.amazonaws.com/stage`
+>
+
+With custom domain names, you can set up your API's hostname, and choose a base path (for example, `myservice`) to map the alternative URL to your API. For example, a more user-friendly API base URL can become:
+
+> `https://api.example.com/myservice`
+> 
+
+In order to set up a custom domain name for your PCM installation, please follow **[Setting up custom domain names for HTTP APIs](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-custom-domain-names.html)**
```

Comment:
Shouldn't we actually document the process of setting this up instead of just linking to the docs?